### DISCUSSION TITLE
posts/2023/12/field-essay-7/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#31] [General]
# posts/2023/12/field-essay-7/

https://guozheng.netlify.app/posts/2023/12/field-essay-7/

**Comment by liechi** [comment]
"一直让阿用做饭，我到点便吃是我心中无法忍受的事情。" 

这句话可以改得更顺些。

**Reply by residualsun1** [reply]
看了一下，这是一个病句。如果按照这个语句顺序写，正确写法应该是“‘一直让阿用做饭，我到点便吃’是我心中无法忍受的事情。”

不过这好像也只是符合主谓宾的格式，看起来怪怪的，只能完全改掉。另，感觉最后一句的”的“应该是”得“哈哈。

**Reply by liechi** [reply]
坦率讲，留言里这个版本似比文章里那个还要顺些。脑补了一个，仅供参考：

>这个中午我也真的做饭了——以往总是阿用做、我吃，时间一长，实在不好意思。另者，既是我有求于人，更没理由被如此照顾了。

**Reply by CyrusYip** [reply]
> ‘一直让阿用做饭，我到点便吃’是我心中无法忍受的事情。（第一版）

我觉得奇怪的点是句子太长了。主语那句话可以单独拿出来。

> 一直让阿用做饭，我到点便吃。这是我心中无法忍受的事情。（修改版）

「事情」和「心中」可以略去。

> 一直让阿用做饭，我到点便吃。我无法忍受这样。（修改版）

---

> 这个中午我也真的做饭了，总是等着阿用做饭给我吃，这是我无法忍受的事情。（第二版）

第二版后面已经很短了，可以合并。

> 这个中午我也真的做饭了，我无法忍受总是等着阿用做饭给我吃。（修改版）

**Reply by residualsun1** [reply]
> 这个中午我也真的做饭了——以往总是阿用做、我吃，时间一长，实在不好意思。另者，既是我有求于人，更没理由被如此照顾了。

这个一看就很有列弛你的个人文风，有点我看民国作家文字的感觉。

> 一直让阿用做饭，我到点便吃。我无法忍受这样。

> 这个中午我也真的做饭了，我无法忍受总是等着阿用做饭给我吃。

我觉得你们改的都比我写的要好很多很多。不知道咋回事，最近我写的文字好像越来越粗糙🫣。

**Reply by liechi** [reply]
改比写要容易些，有旁观者清的便利。

**Reply by liechi** [reply]
> 另，感觉最后一句的”的“应该是”得“哈哈。

对。改了。
